apps/mark/backend/src/util/print_ballot.tsx

```diff
@@ -0,0 +1,62 @@
+import { PrintSides, Printer, renderToPdf } from '@votingworks/printing';
+import { LanguageCode, UiStringsPackage, VotesDict } from '@votingworks/types';
+
+import { assertDefined } from '@votingworks/basics';
+import {
+  BmdPaperBallot,
+  BackendLanguageContextProvider,
+} from '@votingworks/ui';
+import { randomBallotId } from '@votingworks/utils';
+import { Store } from '../store';
+
+export interface PrintBallotProps {
+  printer: Printer;
+  store: Store;
+  precinctId: string;
+  ballotStyleId: string;
+  votes: VotesDict;
+  languageCode: LanguageCode;
+}
+
+export async function printBallot({
+  printer,
+  store,
+  precinctId,
+  ballotStyleId,
+  votes,
+  languageCode,
+}: PrintBallotProps): Promise<void> {
+  const electionDefinition = assertDefined(store.getElectionDefinition());
+  const isLiveMode = !store.getTestMode();
+
+  const uiStringsPackage: UiStringsPackage = {};
+  for (const language of store.getUiStringsStore().getLanguages()) {
+    uiStringsPackage[language] = assertDefined(
+      store.getUiStringsStore().getUiStrings(language)
+    );
```

Comment:
@kofi-q question about this assert - if `getLanguages` returns a language code, does than mean we have `UiStrings` for that language? Or might it return "English" but not have overridden English strings, because that's our default language. Affects whether this assert is safe or not.

Alternatevely, could add a method to the UiStringsStore to get the whole package rather than reconstructing it.

Reply:
IMO, this does feel like the kind of thing the `UiStringsStore` should be able to do and is not unique to printing a ballot.

Reply:
Good question. This assertion makes sense - we only add languages when loading strings for that language. I do like the idea of a `getAllUiStrings`-type store method, now that you mention it, but whatever's simpler - can't think of other use cases for it at this point

Reply:
Helper added.